src/Microsoft.SqlTools.ManagedBatchParser/Microsoft.SqlTools.ManagedBatchParser.csproj

```diff
@@ -4,21 +4,36 @@
 		<TargetFrameworks>net6.0;net472</TargetFrameworks>
 		<LangVersion>9.0</LangVersion>
 		<Nullable>disable</Nullable>
-		<GenerateAssemblyInfo>false</GenerateAssemblyInfo>
+		<AssemblyName>Microsoft.SqlTools.ManagedBatchParser</AssemblyName>
+		<Guid>82dd9738-2ad3-4eb3-9f80-18b594e03621</Guid>
+		<DelaySign>True</DelaySign>
+		<!-- Explicitly disable since it leads to compilation errors. The .NET 6.0 target is used in tests with internalsVisibleTo attribute.-->
+		<SignAssembly Condition="$(TargetFramework) == 'net472'">True</SignAssembly>
+		<AssemblyOriginatorKeyFile>$(RootDir)\SqlToolsKeyPair.snk</AssemblyOriginatorKeyFile>
```

Comment:
Where did you get this from? Make sure you're using the SQL 2003 one, that's what the ESRP service will expect. 

I would also check in the key file - otherwise won't this fail for people locally since the key file doesn't exist?

Reply:
I've added the file too in root folder, its generated using sn.exe. 
I'm not sure if it needs to be a particular snk, since I've not used the ESRP delay sign before, but I'll let build fail and will get the other one if we need that one.

Reply:
I'm fairly certain there's a small number of valid public key tokens ESRP will sign.